Static code check (gometalinter) fails

[cni-plugin] Running shell script
+ make static-checks test-containerized
docker run --rm \
	-e LOCAL_USER_ID=1000 \
	-v /home/jenkins/workspace/calico-cni-pipeline/cni-plugin:/go/src/github.com/projectcalico/cni-plugin \
	calico/go-build sh -c '\
		cd  /go/src/github.com/projectcalico/cni-plugin && \
		gometalinter --deadline=300s --disable-all --enable=goimports --enable=vet --enable=errcheck --vendor -s test_utils ./...'
Starting with UID : 1000
calico_cni_k8s_test.go:199:20:warning: error return value not checked (CreateHostVeth("", name, K8S_TEST_NS)) (errcheck)
calico_cni_test.go:159:20:warning: error return value not checked (CreateHostVeth(container_id, "", "")) (errcheck)
Makefile:179: recipe for target 'static-checks' failed
make: *** [static-checks] Error 1

--- a/calico_cni_k8s_test.go
+++ b/calico_cni_k8s_test.go
@@ -196,7 +196,9 @@ var _ = Describe("CalicoCni", func() {
 					})
 					Expect(err).NotTo(HaveOccurred())
 
-					CreateHostVeth("", name, K8S_TEST_NS)
+					if err := CreateHostVeth("", name, K8S_TEST_NS); err != nil {
+						panic(err)
+					}
 					_, netnspath, session, _, _, _, err := CreateContainer(netconf, name, "")
 					Expect(err).ShouldNot(HaveOccurred())
 					Eventually(session).Should(gexec.Exit(0))

--- a/calico_cni_test.go
+++ b/calico_cni_test.go
@@ -156,7 +156,9 @@ var _ = Describe("CalicoCni", func() {
 			Context("when the same hostVeth exists", func() {
 				It("successfully networks the namespace", func() {
 					container_id := fmt.Sprintf("con%d", rand.Uint32())
-					CreateHostVeth(container_id, "", "")
+					if err := CreateHostVeth(container_id, "", ""); err != nil {
+						panic(err)
+					}
 					_, netnspath, session, _, _, _, err := CreateContainerWithId(netconf, "", "", container_id)
 					Expect(err).ShouldNot(HaveOccurred())
 					Eventually(session).Should(gexec.Exit(0))

